src/en/news/blog/2023/v18-2-0-reef-released/index.md

```diff
@@ -0,0 +1,953 @@
+      mgr: foo(active), standbys: bar, baz
+      ...
+      ```
+
+4. Upgrade all OSDs by installing the new packages and restarting the ceph-osd daemons on all OSD hosts
+
+      ```
+      systemctl restart ceph-osd.target
+      ```
+      
+5. Upgrade all CephFS MDS daemons. For each CephFS file system,
+
+   1. Disable standby_replay:
+
+         ```
+         ceph fs set <fs_name> allow_standby_replay false
+         ```
+
+   2. **Only** if upgrading from a Pacific <=16.2.5:
+
+         ```
+         ceph config set mon mon_mds_skip_sanity true
+         ```
+
+   3. Reduce the number of ranks to 1. (Make note of the original number of MDS daemons first if you plan to restore it later.)
+
+         ```
+         ceph status # ceph fs set <fs_name> max_mds 1
+         ```
+
+   4. Wait for the cluster to deactivate any non-zero ranks by periodically checking the status
+
+         ```
+         ceph status
+         ```
+
+   5. Take all standby MDS daemons offline on the appropriate hosts with
+
+         ```
+         systemctl stop ceph-mds@<daemon_name>
+         ```
+
+   6. Confirm that only one MDS is online and is rank 0 for your FS
+
+         ```
+         ceph status
+         ```
+
+   7. Upgrade the last remaining MDS daemon by installing the new packages and restarting the daemon
+
+         ```
+         systemctl restart ceph-mds.target
+         ```
+
+   8. Restart all standby MDS daemons that were taken offline
+
+         ```
+         systemctl start ceph-mds.target
+         ```
+
+   9. Restore the original value of `max_mds` for the volume
+
+         ```
+         ceph fs set <fs_name> max_mds <original_max_mds>
+         ```
+
+6. Upgrade all radosgw daemons by upgrading packages and restarting daemons on all hosts
+
+      ```
+      systemctl restart ceph-radosgw.target
+      ```
+
+7. Complete the upgrade by disallowing pre-Reef OSDs and enabling all new Reef-only functionality
+
+      ```
+      ceph osd require-osd-release reef
+      ```
+
+8. If you set `noout` at the beginning, be sure to clear it with
+
+      ```
+      ceph osd unset noout
+      ```
+
+9. Consider transitioning your cluster to use the cephadm deployment and orchestration framework to simplify cluster management and future upgrades. For more information on converting an existing cluster to cephadm, see https://docs.ceph.com/en/reef/cephadm/adoption/.
+
+### Post-upgrade
+
+1. Verify the cluster is healthy with `ceph health`. If your cluster is running Filestore, and you are upgrading directly from Pacific to Reef, a deprecation warning is expected. This warning can be temporarily muted using the following command
+
+      ```
+      ceph health mute OSD_FILESTORE
+      ```
+
+2. If you are upgrading from Mimic, or did not already do so when you upgraded to Nautilus, we recommend you enable the new [v2 network protocol \<msgr2\>](https://docs.ceph.com/en/reef/rados/configuration/msgr2/), issue the following command
```

Comment:
Is this all still relevant?

Reply:
No, just confirmed w/ Radek that it should be removed.